scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -5,11 +5,11 @@ export CLUSTER_NAME=${CLUSTER_NAME:-"test1"}
 
 export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
 export IMAGE_USERNAME=${IMAGE_USERNAME:-"metal3"}
-export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.18.0"}
-export KUBERNETES_BINARIES_VERSION=${KUBERNETES_BINARIES_VERSION:-"v1.18.0"}
-export UPGRADED_K8S_VERSION_1=${UPGRADED_K8S_VERSION_1:-"v1.18.1"}
-export UPGRADED_K8S_VERSION_2=${UPGRADED_K8S_VERSION_2:-"v1.18.2"}
-export UPGRADED_BINARY_VERSION=${UPGRADED_BINARY_VERSION:-"v1.18.1"}
+export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.18.8"}
+export KUBERNETES_BINARIES_VERSION=${KUBERNETES_BINARIES_VERSION:-"v1.18.8"}
+export UPGRADED_K8S_VERSION_1=${UPGRADED_K8S_VERSION_1:-"v1.18.9"}
+export UPGRADED_K8S_VERSION_2=${UPGRADED_K8S_VERSION_2:-"v1.19.0"}
```

Comment:
you should verify that this is supported afaik v0.3.9 was not supporting upgrades to 1.19 . But maybe 0.3.10 does ?

Reply:
I stepped down one patch version, v1.18.7 to v1.18.9 now used

Reply:
1.18.9 used instead